src/dbally/__init__.py

```diff
@@ -21,29 +23,32 @@
 from .exceptions import DbAllyError
 from .llms.clients.exceptions import LLMConnectionError, LLMError, LLMResponseError, LLMStatusError
 
+global_event_handlers: List[Callable] = []
```

Comment:
Let's name it just `event_handlers`

Reply:
I called it this way to not overwrite it by accident by local variable.
Regarding overwriting, collection module event_handler is also name of the ask parameter. 
When we use global with prefix dbally.event_handler overwrite would not happens but I thought it could be tricky if someone won't refer to it like that.

Reply:
Maybe we should rename the local variable to something like `override_event_handlers`? That way, it's clear that we're providing handlers meant to override the existing ones.